include/nbl/system/CFileWin32.h

```diff
@@ -1,5 +1,5 @@
-#ifndef	CFILE_WIN32_H
-#define	CFILE_WIN32_H
+#ifndef	_NBL_SYSTEM_CFILEWIN32_H_INCLUDED_
+#define	_NBL_SYSTEM_CFILEWIN32_H_INCLUDED_
```

Comment:
double `_` on both sides

Reply:
I believe thats reserved in the C preprocessor

Reply:
well, we do this all over the engine XD

Reply:
point is, we probably shouldn't XD